web-quiz-app/src/components/TextQuestion.jsx

```diff
@@ -30,12 +70,22 @@ function TextQuestion() {
     }
 
     function submit() {
+        QuizDataService.crossCheckQuizID(JSON.parse(localStorage.getItem('lastSelectedQuiz')), userID)
+        .then(response => {
+            setQuizID(response.data)
+        })
+        .catch(e =>{
+            console.log(e)
+        })
+    }
+
+    function createQuestion() {
+        console.log(quizID)
         var questionToCreate = {
-            id: 123,
             question_data: 1,
-            question_text: question,
-            question_total_marks: 10,
-            quiz_id: 1,
+            question_text: question + "|||" + solution,

```

Comment:
As a note, ||| is currently used for multiple choice questions to indicate the possible options for a question (e.g. option A, option B...)

Is it at all possible to put the solution in a separate column in the database?

Reply:
Right, I didn't know the structure of how you were doing the responses so this was just a guess from what I remember seeing. It can definitely be changed and was something I forgot to ask you about

Reply:
Great. I think if we edit it like this for now, we can merge this in and address the solution column separately. 
```suggestion
            question_text: question,

```